Eslint reports an error and warnings

```
$ npm run lint 

> screeps-arena-typescript-starter@3.0.0 lint
> eslint "src/**/*.ts"


/home/somq14/workspace/screeps-arena-typescript-starter/src/arena_capture_the_flag/main.ts
  32:10  warning  'BodyPart' is defined but never used  @typescript-eslint/no-unused-vars

/home/somq14/workspace/screeps-arena-typescript-starter/src/utils/ErrorMapper.ts
   8:9   warning  Unexpected dangling '_' in '_consumer'       no-underscore-dangle
   9:7   warning  Unexpected dangling '_' in '_consumer'       no-underscore-dangle
  12:12  warning  Unexpected dangling '_' in '_consumer'       no-underscore-dangle
  82:11  error    Expected space or tab after '//' in comment  spaced-comment

✖ 5 problems (1 error, 4 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -34,6 +34,7 @@ module.exports = {
       "game/utils",
       "game/path-finder",
       "game/constants",
+      "game/visual",
       "arena",
       "arena/prototypes",
       "arena/constants"

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -78,7 +78,7 @@ module.exports = {
     "no-invalid-this": "off",
     "no-multiple-empty-lines": "off",
     "no-new-wrappers": "error",
-    "no-shadow": [
+    "@typescript-eslint/no-shadow": [
       "error",
       {
         hoist: "all"

--- a/package.json
+++ b/package.json
@@ -38,7 +38,7 @@
     "@typescript-eslint/parser": "^5.18.0",
     "@typescript-eslint/typescript-estree": "^5.18.0",
     "chai": "^4.2.0",
-    "eslint": "^6.8.0",
+    "eslint": "^8.13.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-import-resolver-typescript": "^2.0.0",
     "eslint-plugin-import": "^2.22.0",

--- a/src/arena_capture_the_flag/main.ts
+++ b/src/arena_capture_the_flag/main.ts
@@ -29,11 +29,11 @@
 
 // This stuff is arena-specific
 import { ATTACK, HEAL, RANGED_ATTACK } from "game/constants";
-import { BodyPart, Flag } from "arena";
 import { Creep, GameObject } from "game/prototypes";
 import { getDirection, getObjectsByPrototype, getRange, getTicks } from "game/utils";
-import { searchPath } from "game/path-finder";
+import { Flag } from "arena";
 import { Visual } from "game/visual";
+import { searchPath } from "game/path-finder";
 
 declare module "game/prototypes" {
   interface Creep {

--- a/src/arena_capture_the_flag/main.ts
+++ b/src/arena_capture_the_flag/main.ts
@@ -92,9 +92,9 @@ function meleeAttacker(creep: Creep) {
       font: "0.5",
       opacity: 0.7,
       backgroundColor: "#808080",
-      backgroundPadding: 0.03,
+      backgroundPadding: 0.03
     }
-    );
+  );
   const targets = enemyCreeps
     .filter(i => getRange(i, creep.initialPos) < 10)
     .sort((a, b) => getRange(a, creep) - getRange(b, creep));

